server/scripts/start_chain.sh

```diff
@@ -92,27 +82,25 @@ i=1;
 m=""
 while [ ${i} -le ${N} ]
 do
-    m="${m} --member \$(memcoin --config /tmp/${S}${i} ordering export)"
+    m="${m} --member \$(chaincli --config /tmp/${W}${i} ordering export)"
     i=$((i + 1));
 done
-# sent to master pane
-tmux send-keys -t ${S}:0.%0 "memcoin --config /tmp/${S}1 ordering setup ${m}" C-m
+tmux send-keys -t ${S}:${W}.0 "chaincli --config /tmp/${W}1 ordering setup ${m}" C-m
 
 
 echo -e "${GREEN}[ACCESS]${NC} setup access rights on each node"
 i=1;
 while [ ${i} -le ${N} ]
 do
-    # sent to master pane
-    tmux send-keys -t ${S}:0.%0 "memcoin --config /tmp/${S}${i} access add \
+    tmux send-keys -t ${S}:${W}.0 "chaincli --config /tmp/${W}${i} access add \
                                   --identity $(crypto bls signer read --path private.key --format BASE64_PUBKEY)" C-m
     i=$((i + 1));
 done
 
 
 echo -e "${GREEN}[GRANT]${NC} grant access for node 1 on the chain"
 # sent to master pane
-tmux send-keys -t ${S}:0.%0 "memcoin --config /tmp/${S}1 pool add\
+tmux send-keys -t ${S}:${W}.0 "chaincli --config /tmp/${W}1 pool add\
     --key private.key\
     --args go.dedis.ch/dela.ContractArg --args go.dedis.ch/dela.Access\
     --args access:grant_id --args 0200000000000000000000000000000000000000000000000000000000000000\
```

Comment:
need to grant 
```
--args access:grant_contract --args go.dedis.ch/calypso.SMC \
```
instead of
```
--args access:grant_contract --args go.dedis.ch/dela.Value \
```